Product/Production/pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>Adapters</module>
         <module>Gateway</module>
         <module>Deploy</module>
+		<module>../../../Plugins/PerformanceTestTools</module>
```

Comment:
Plugins module can be built separately. Do we need to include it in CONNECT build?